docker-compose.yml

```diff
@@ -94,7 +94,7 @@ services:
 
   proxy:
     # https://github.com/silinternational/traefik-https-proxy
-    image: silintl/traefik-https-proxy
+    image: silintl/traefik-https-proxy:latest
```

Comment:
Any reason your net setting an explicit version here?

Reply:
It's not available yet. [CVR-885](https://itse.youtrack.cloud/issue/CVR-885) push version tags for silintl/traefik-https-proxy